core/cmd/app.go

```diff
@@ -56,9 +56,18 @@ func NewApp(client *Client) *cli.App {
 			Usage:  "EXPERIMENTAL: TOML configuration file via flag, or raw TOML via env var. If used, legacy env vars must not be set.",
 			EnvVar: "CL_CONFIG",
 		},
+		cli.StringFlag{
+			Name:  "secrets, s",
+			Usage: "EXPERIMENTAL: TOML configuration file for secrets. Must be set if and only if config is set.",
+		},
 	}
 	app.Before = func(c *cli.Context) error {
 		if c.IsSet("config") {
+			if !c.IsSet("secrets") {
```

Comment:
This seems extreme. Why is it required?

Reply:
Yeah no need to panic, and we also plan to support env vars as an alternative, right?